Applications/demos/rocket_test/assets/header.wns

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Why the empty file?

Reply:
This was mostly for testing. (rocket_test is going to go away completely once engine is up). But this is `#included` by the main.rml. Just to make sure that this works.